src/common/xmlhelpers.cpp

```diff
@@ -389,6 +389,7 @@ namespace odata { namespace edm {
 #else // LINUX
         xmlChar *memory;
         int size;
+	xmlTextWriterFlush(m_writer); 
```

Comment:
Please fix the indentation in this file. We commonly use four-space soft tab. Thanks.